openjdk8/src/main/docker/Dockerfile

```diff
@@ -12,12 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gcr.io/google_appengine/base
+FROM ${docker.appengine.image}
```

Comment:
What's the motivation for externalizing this into a property? If it's only used here, I'd rather avoid the extra level of indirection.